PopApp/App.js

```diff
@@ -1,15 +1,15 @@
-import 'react-native-gesture-handler'
```

Comment:
The `import 'react-native-gesture-handler'` is compulsory to use react-navigation see [here](https://reactnavigation.org/docs/getting-started)

Reply:
Indeed my bad! Added it back.

I will remember next time to double check before deleting an import that IntelliJ deems useless! ^^'